static/css/components/read-panel.less

```diff
@@ -29,6 +29,32 @@
     margin-bottom: 10px;
   }
 
+  // Preview button appears styled as a text link with a book icon
+  .cta-btn--preview {
+    color: @primary-blue;
+    margin: -5px auto 15px;
+    padding: 5px 20px 10px; // Increasing click area for mobile text link
+    text-decoration: none;
+    position: relative;
+
+    &::before {
+      width: 22px;
+      height: 18px;
+      display: inline-block;
+      margin-right: 6px;
+      position: relative;
+      top: 2px;
+
+      // Setting pseudo-element content to book icon; mask is used to allow for a 'fill' color
+      content: '';
+      -webkit-mask: url(../images/icons/open-book.svg);
+      mask: url(../images/icons/open-book.svg);
+      -webkit-mask-size: cover;
+      mask-size: cover;
+      background-color: @primary-blue;
+    }
+  }
```

Comment:
I think that the preview button styling should be in `buttonCta.less` instead of `read-panel.less`.  Sorry that I didn't notice this sooner.

Reply:
No problem! I'll move this chunk over there.